resources/lang/en/beatmapset_events.php

```diff
@@ -27,6 +27,7 @@
         'nominate' => 'Nominated by :user.',
         'nominate_modes' => 'Nominated by :user (:modes).',
         'nomination_reset' => 'New problem :discussion (:text) triggered a nomination reset.',
+        'offset_edit' => 'Online offset changed from :old to :new.',
```

Comment:
also need translation for `beatmapset_events.type.offset_edit` (used in `/beatmapsets/events`)